modules/backend/widgets/MediaManager.php

```diff
@@ -1472,24 +1470,11 @@ protected function getTargetDimensions($width, $height, $originalImagePath)
     }
 
     /**
-     * Detect the upload post flag
-     * @return void
+     * Upload handler for the server-side processing of uploaded files
      */
-    protected function checkUploadPostback()
+    public function onUpload()
     {
-        if ($this->readOnly) {
-            return;
-        }
```

Comment:
This check needs to stay here @alxy

Reply:
I would say we don't need this any longer, as the handler is no longer ran on any initialization (constructor) of the widget. But I will investigate this next weekend.